dnf5/commands/check-upgrade/check-upgrade.cpp

```diff
@@ -46,6 +46,17 @@ void CheckUpgradeCommand::set_argument_parser() {
     auto & cmd = *get_argument_parser_command();
     cmd.set_description("Check for available package upgrades");
 
+    minimal = dynamic_cast<libdnf5::OptionBool *>(
+        parser.add_init_value(std::unique_ptr<libdnf5::OptionBool>(new libdnf5::OptionBool(false))));
+    auto minimal_opt = parser.add_new_named_arg("minimal");
+    minimal_opt->set_long_name("minimal");
+    minimal_opt->set_description(
+        "Reports the lowest versions of packages that fix a bugfix, enhancement or a fix for a security issue "
+        "(security) on the system.");
```

Comment:
Both this description and the man page don't mention the `newpackage` advisory type which is also used.

What about to change it a bit to: `Reports the lowest versions of packages that fix advisories of type bugfix, enhancement, security, or newpackage.`

Anyway I do like your description much more than the one in `upgrade` command which is quite ambiguous. Could they be unified? [There is even a TODO for it.](https://github.com/rpm-software-management/dnf5/blob/main/dnf5/commands/upgrade/upgrade.cpp#L47)